### DISCUSSION TITLE
GIF Background

### DISCUSSION THREAD

**Discussion by bennoah1216** [hugo-themes/toha#500] [Q&A]
I added a GIF as the background of my website (you can see a draft [here](www.bennoah1216.github.io)), but when the window gets resized too small it freezes as a static image. Is there any way to change this?

Thanks!

Ben

**Comment by hossainemruz** [comment]
I can confirm the behavior. I am not sure why this is happening. The theme resize the background for smaller screen. That might cause the issue.

**Comment by bennoah1216** [comment]
Sorry just wanted to check again to see if anyone has figured out a solution.
